docs/introduction.mdx

```diff
@@ -20,25 +20,39 @@ and [any datastore you can build a plugin for](/docs/connectors/building).
 It's written in [Go](https://go.dev/), compiles to a binary, and is designed to
 be easy to use and [deploy](/docs/running/binary).
 
+Out of the box, Conduit comes with:
+
+- A UI
+- Common connectors 
+- Processors
+- Observability
+
 In this getting started guide we'll use a pre-built binary, but Conduit can also be run using [Docker](/docs/running/docker).
 
-## Installing
+### It’s simple, yet powerful
 
-To get started:
+Eliminate the multi-step process you go through today. Just download the binary
+and start building. Conduit pipelines run in their own goroutines and are
+connected using Go channels. This makes Conduit pipelines incredibly performant
+on multi-core machines.
 
-1. [Download the latest Conduit release](https://github.com/ConduitIO/conduit/releases/latest).
-2. Unzip:
+### It’s real-time
 
-If you're on Mac, it will look something like this:
+Conduit pipelines listen for changes to a database, data warehouse, etc., and
+allows your data applications to act upon those changes in real-time.
 
-```shell
-tar zxvf conduit_0.10.0_Darwin_x86_64.tar.gz
-```
+### It’s extensible
 
-Depending on your operating system you may need to make the conduit binary executable.
-```shell
-chmod a+x conduit
-```
+Conduit connectors are plugins that communicate with Conduit via a gRPC
+interface. This means that plugins can be written in any language as long as
+they conform to the required interface. Check out
+our [connector docs](/docs/connectors/installing)!
+
+## Installing
+
+To get started make sure you download the latest version of Conduit using one of our available options.
```

Comment:
I would provide a snippet how to install it via homebrew and then refer to the install page for other methods, otherwise the reader needs to navigate to a different page, which kind of breaks the flow.